js/siteorigin-panels/view/live-editor.js

```diff
@@ -19,7 +20,7 @@ module.exports = Backbone.View.extend( {
      * Render the live editor
      */
     render: function(){
-        this.$el.html( this.template() );
+        this.setElement( this.template() );
```

Comment:
I think this was so we could have the wrapper in the actual template instead. Having a set, named wrapper was important for how we're moving the builder in and out of the live editor sidebar.